src/layout/SigningStatusPanel/config.ts

```diff
@@ -0,0 +1,161 @@
+import { CG } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Presentation,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+    renderInTabs: false,
```

Comment:
I don't see why this couldn't render in Accordion, Cards and Tabs? :thinking: Is is just because we haven't tested that, or because we want this to strictly not render inside anything else? I believe opening these up for everything that's not obviously breaking stuff is the best. (Such as rendering in `AccordionGroup` or `ButtonGroup`, or a table cell for that matter.) That way, app developers will get the final say instead of us. We've had pull requests from external contributors before with one-line changes that only allow one type of component to be rendered inside another, and at that point we're just slowing people down by being gatekeepers.